firmware/Core/Src/unit_tests/test_telecommand_arg_helpers.c

```diff
@@ -1,6 +1,35 @@
 #include "telecommands/telecommand_args_helpers.h"
 #include "unit_tests/unit_test_helpers.h"
 
+uint8_t TEST_EXEC__TCMD_ascii_to_uint64() {
+    uint64_t result;
+
+    // Success: Nominal test
+    TEST_ASSERT(TCMD_ascii_to_uint64("123", 3, &result) == 0);
+    TEST_ASSERT(result == 123);
+
+    // Error: Empty string
```

Comment:
The maximum value of a `uint64_t` is `18446744073709551615` (herein `UINT64_MAX`). Test cases for `UINT64_MAX`, `UINT64_MAX+1`, and `UINT64_MAX-1` would be valuable.

I believe this function currently overflows if you pass it `UINT64_MAX-1` (or any other 20-digit number larger than this max). 

I believe setting the max "digit length" to 19 characters would be appropriate (even though it does prevent passing the UINT64_MAX as a max value). A comment in the docstring explaining the limitation would mitigate this restriction.

Reply:
I updated so that the max digit length is 19 characters and added a note describing the restriction in the docstring.